ui/components/multichain/app-header/app-header.scss

```diff
@@ -33,8 +33,8 @@
       max-width: 250px;
     }
 
-    &--avatar-network {
-      padding: 0; // TODO: Remove once https://github.com/MetaMask/metamask-extension/pull/17006 is merged
+    &--avatar-network p {
+      display: none;
```

Comment:
So I think this will have some accessibility concerns. But I see that `label` is a required prop for PickerNetwork I think we may need to adjust the `PickerNetwork` component slightly.

Reply:
Is it fine that we resolved it with aria-label now or does this need any update in Picker Network as well?

Reply:
I'm actually not sure but I think the aria-label should be fine 👍